lz4/frame/__init__.py

```diff
@@ -25,9 +25,9 @@
 __doc__ = _doc
 
 try:
-    import _compression   # Python 3.6 and later
+    import compression._common._streams as _compression  # Python 3.14
 except ImportError:
-    from . import _compression
+    import _compression   # Python 3.6 - 3.13
```

Comment:
```suggestion
    import _compression   # Python 3.9 - 3.13
```
https://github.com/python-lz4/python-lz4/blob/022d53815abf97e6eb49d0be3a27c74541c782ec/setup.py#L174